cfpb/cfgov-refresh
# Paying for College - Add program salaries to front-end

This pull request adds program selection and handling of program-specific salaries for graduate students (where data is available). 

## Additions/Changes
- A dropdown menu that appears when school data is available
- Content that appears when a program is selected, or when program data is unavailable
- Fix content for undergraduates with regards to salary figures
- Fix empty/short calls to the API

## How to test this PR
1. Pull down the PR and fire up `cfgov-refresh` locally.
2. Get the latest data loaded in there (`refresh-data.sh`)!
3. navigate to `localhost/paying-for-college/your-financial-path-to-graduation/`
4. Try out different graduate programs. Schools where we have program data (try George Washington University) should allow you to select a program and show a different salary than if you select undergrad. Schools with no program data should inform you of that.

## Checklist
